### ISSUE TITLE
Compilation error with Elixir 1.8.0

### ISSUE THREAD

**Issue by IceDragon200** [bettyblocks/liquid-elixir#91]
While compiling under elixir 1.8.0, the compiler produces the following error

```
== Compilation error in file lib/liquid/filters.ex ==
** (CompileError) lib/liquid/filters.ex:146: imported Kernel.floor/1 conflicts with local function
    (elixir) src/elixir_locals.erl:93: :elixir_locals."-ensure_no_import_conflict/3-lc$^0/1-0-"/2
    (elixir) src/elixir_locals.erl:94: anonymous fn/3 in :elixir_locals.ensure_no_import_conflict/3
    lib/liquid/filters.ex:9: (module)
```

**Comment by storestartup** [comment]
Elixir 1.8 seems to have introduced Kernel.floor/1 and Kernel.ceil/1 which conflict with function definitions in Liquid.Filters
see #92

**Comment by nulian** [comment]
Should be fixed in 0.9.0
